scripts/tests/files/extreme_chr_mir_anno.gff3

```diff
@@ -1,3 +1,2 @@
-19	.	miRNA	0	80	.	+	.	ID=MIMAT0002822;Alias=MIMAT0002822;Name=hsa-miR-512-5p;Derives_from=MI0003140
-19	.	miRNA	83	124	.	+	.	ID=MIMAT0002823;Alias=MIMAT0002823;Name=hsa-miR-512-3p;Derives_from=MI0003140
+19	.	miRNA	1	80	.	+	.	ID=MIMAT0002822;Alias=MIMAT0002822;Name=hsa-miR-512-1-5p;Derives_from=MI0003140
```

Comment:
By setting this to 1 (which is legal) instead of 0 (which is illegal) here an in the next file - are you still testing what happens when a GFF annotation has a 0 (illegal) coordinate?

Reply:
I added a `try-except` block in the local database creation method `set_db` that raises the custom annotation exception and its correspondent unit test and file. This way, as we are not giving the possibility of setting a starting position at 0, if the initial file is wrong, the whole script stops.